### DISCUSSION TITLE
Why is wayland required to build? Is there a flag to disable it during compilation?

### DISCUSSION THREAD

**Discussion by mobius3** [wezterm/wezterm#1455] [Q&A] [answered]
Hi. I'm using Gentoo and I have no plans whatsoever of using wayland, not right now at least. I'm trying to compile from source, to potentially maybe create an ebuild for it, but I can't because of the missing wayland dependency.

Is there some sort of flag that I can pass that would disable wayland dependency altogether?

**Comment by wez** [comment, marked as answer]
You can run `cargo build --release --no-default-features` to build without wayland support.
This was added in 8ab025b7fc5e4361c9e5f952a6977d35518cb802

**Reply by mobius3** [reply]
Thanks!

**Reply by mobius3** [reply]
Maybe this should be documented in the install docs?

**Reply by wez** [reply]
Yeah, c96c36fb6 adds a note about that
